onnxruntime/test/onnx/TestResultStat.h

```diff
@@ -33,12 +35,27 @@ class TestResultStat {
     failed_kernels.insert(s);
   }
 
+  struct OrderPair {
+    bool operator () (const std::pair<std::string, std::string>& p1, const std::pair<std::string, std::string>& p2) const {
```

Comment:
>operator ()  [](start = 9, length = 12)

This is operator < not function call operator.
Also I think std::pair has that operator already defined.
for std::string it is imlemented using std::string::compare if I am not mistaken